### PR TITLE
Add ability to set recommended maps and mods

### PR DESCRIPTION
**Pull request by Sheikah45**
tied to https://github.com/FAForever/faf-java-api/pull/472

### CODE REVIEW

**Comment by Sheikah45** [comment]
@Brutus5000 don't have access to request reviews on this one

File: src/main/java/com/faforever/moderatorclient/ui/ViewHelper.java [hunk 1]
```diff
@@ -695,6 +678,13 @@ public static void buildModTableView(TableView<ModFX> tableView, ObservableList<
         tableView.getColumns().add(authorColumn);
         extractors.put(authorColumn, modFX -> modFX.getUploader().getLogin());
 
+        TableColumn<ModFX, Boolean> recommendColumn = new TableColumn<>("Recommended");
+        recommendColumn.setCellValueFactory(o -> o.getValue().recommendedProperty());
+        recommendColumn.setCellFactory(CheckBoxTableCell.forTableColumn(recommendColumn));
+        recommendColumn.setMinWidth(100);
+        tableView.getColumns().add(recommendColumn);
+        extractors.put(recommendColumn, ModFX::isRecommended);
```
Comment (Brutus5000) [inline src/main/java/com/faforever/moderatorclient/ui/ViewHelper.java]: I don't think we want to offer copying boolean values via context menu. Please remove the extractors.put

File: src/main/java/com/faforever/moderatorclient/ui/ViewHelper.java [hunk 2]
```diff
@@ -651,6 +627,13 @@ public static void buildMapTableView(TableView<MapFX> tableView, ObservableList<
         tableView.getColumns().add(authorColumn);
         extractors.put(authorColumn, mapFX -> mapFX.getAuthor().getLogin());
 
+        TableColumn<MapFX, Boolean> recommendColumn = new TableColumn<>("Recommended");
+        recommendColumn.setCellValueFactory(o -> o.getValue().recommendedProperty());
+        recommendColumn.setCellFactory(CheckBoxTableCell.forTableColumn(recommendColumn));
+        recommendColumn.setMinWidth(100);
+        tableView.getColumns().add(recommendColumn);
+        extractors.put(recommendColumn, MapFX::isRecommended);
```
Comment (Brutus5000) [inline src/main/java/com/faforever/moderatorclient/ui/ViewHelper.java]: I don't think we want to offer copying boolean values via context menu. Please remove the extractors.put

**Review by Brutus5000 — CHANGES_REQUESTED** [review]
Please stop converting everything into * imports

**Comment by Sheikah45** [comment]
Sigh the intellij auto optimization is changing to * guess I will have to turn that off

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**